samples/LocalEnvSecretStoreSample/LocalEnvSecretStoreSample.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Dapr.PluggableComponents.AspNetCore" Version="0.2.0" />

```

Comment:
This isn't needed as the project is already referencing the `AspNetCore` project directly.